api/types/types.go

```diff
@@ -387,6 +387,7 @@ type NetworkResource struct {
 	Containers map[string]EndpointResource // Containers contains endpoints belonging to the network
 	Options    map[string]string           // Options holds the network specific options to use for when creating the network
 	Labels     map[string]string           // Labels holds metadata specific to the network being created
+	Peers      []network.PeerInfo          // List of peer nodes for an overlay network
```

Comment:
I think it is reasonable to add `json:",omitempty"` to this (since it make sense only for global-scoped networks.